test/integration/backward_compatible/sql/ExecuteTest.js

```diff
@@ -450,49 +450,11 @@ describe('SqlExecuteTest', function () {
         });
     });
     describe('errors/invalid usage', function () {
-        const LITE_MEMBER_CONFIG = `
```

Comment:
Instead of deleting them, we can set the maximum version of the client that this test would run to 4.2 so that we test the behavior for it.

Also, can you add a test with a cluster that has lite and data members, do some queries and make sure that it works for Node.js client as well?

Reply:
I realized that the test is valid for 5.0 and 4.x. I just deleted detailed error code and originatedMemberId checks to make it pass for both. Is it ok like this or should I add conditional checks? I think it is ok like this.

I added the test you mentioned